plugins/osc/OSCNode.cpp

```diff
@@ -150,7 +150,7 @@ int OSCDataHandler(const char *osc_address, const char *types, lo_arg **argv,
 
   if (argc == 1) {
     if (type == "b") {
-      lo_blob blob = argv[0];
+      lo_blob blob = (lo_blob)argv[0];
```

Comment:
Example code in liblo also uses a regular typecast:

https://github.com/radarsat1/liblo/blob/c1a51bca21e8535ce77a9daf256f2e74c1a7e80f/examples/example_server.c#L125